R/createYaml.R

```diff
@@ -0,0 +1,161 @@
+#' Creates a yaml file storing the parameters that are used for creating the PTXQC report, 
+#' optionally returns these parameters and a list of qc-Metrics
+#'
+#' @param yc A yaml class object created by YAMLClass$new()
+#' @param path path to an (empty) yaml file

```

Comment:
describe the WHAT and WHY, i.e. `@param store_path Store the YAML config in this file;  will be overwritten if existing`